morf-core/src/main/java/org/alfasoftware/morf/jdbc/SqlDialect.java

```diff
@@ -1801,6 +1806,17 @@ protected String getSqlFrom(BlobFieldLiteral field) {
     return String.format("'%s'", field.getValue());
   }
 
+  /**
+   * Default implementation will just return the Base64 representation of the binary data, which may not necessarily work with all SQL dialects.
+   * Hence appropriate conversions to the appropriate type based on facilities provided by the dialect's SQL vendor implementation should be used.
+   *
+   * @param field the BLOB field literal
+   * @return the SQL construct or base64 string representation of the binary value
+   */
+  protected String getSqlFrom(ClobFieldLiteral field) {
+    return getSqlFrom((FieldLiteral) field);
+  }

```

Comment:
JavaDoc was probably copied from BLOB method.
Is it actually true, that we use Base64 for CLOBs too?
Probably best to review the whole comment